<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newMeetingPage</class>
 <widget class="QWidget" name="newMeetingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>会议名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>预计参加人数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>预计开始时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>持续时长</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>可预订的会议室：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>会议说明：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="meetingNameLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="attendNumSpinBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="startTime">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="meetTimeComboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>选择会议时长</string>
         </property>
         <item>
          <property name="text">
           <string>15分钟</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30分钟</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1小时</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1小时30分钟</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2小时</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="roomComboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="descriptionTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>选择参会人员：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="departmentComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="unattendListView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QToolButton" name="down_btn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/image_all.qrc">
           <normaloff>:/image/image/down.svg</normaloff>:/image/image/down.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="up_btn">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../res/image_all.qrc">
           <normaloff>:/image/image/up.svg</normaloff>:/image/image/up.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="attendListView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="book_btn">
         <property name="text">
          <string>预订会议</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_btn">
         <property name="text">
          <string>重置</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/image_all.qrc"/>
 </resources>
 <connections/>
</ui>
